index.html

```diff
@@ -15,24 +16,7 @@
       <input id="search" type="text" placeholder="&#128269; Search"></input>
     </div>
     <div class="contact-list">
-      <div class="contact"><p class="name">Andy Mention</p></div>
-      <div class="contact"><p class="name">Emily Davis</p></div>
-      <div class="contact"><p class="name">Eric Fransen</p></div>
-      <div class="contact"><p class="name">Jessica Goulding</p></div>
-      <div class="contact"><p class="name">Jonmichael Chambers</p></div>
-      <div class="contact"><p class="name">Marc Garreau</p></div>
-      <div class="contact"><p class="name">Tan Doan</p></div>
-      <div class="contact"><p class="name">Alan Smith</p></div>
-      <div class="contact"><p class="name">Allison Larson</p></div>
-      <div class="contact"><p class="name">Andrew Watkins</p></div>
-      <div class="contact"><p class="name">Chad Brading</p></div>
-      <div class="contact"><p class="name">Corey Davis</p></div>
-      <div class="contact"><p class="name">Gustavo Villagrana</p></div>
-      <div class="contact"><p class="name">Hilary Denton</p></div>
-      <div class="contact"><p class="name">Horacio Chavez</p></div>
-      <div class="contact"><p class="name">Tim Proctor</p></div>
-      <div class="contact"><p class="name">Will Faurot</p></div>
+      <script>printNames();</script>
```

Comment:
Another thing with jQuery: we can use `document.ready()` to fire off our scripts instead of calling it inline here.